server/src/main/java/io/pravega/schemaregistry/storage/client/TableStore.java

```diff
@@ -417,4 +434,25 @@ private String getToken(String tableName) {
         private final String table;
         private final K key;
     }
+
+    private byte[] getArray(ByteBuf buf) {
+        final byte[] bytes = new byte[buf.readableBytes()];
+        final int readerIndex = buf.readerIndex();
+        buf.getBytes(readerIndex, bytes);
+        return bytes;
+    }
+
+    private void releaseKeys(Collection<TableSegmentKey> keys) {
+        for (TableSegmentKey k : keys) {
+            ReferenceCountUtil.safeRelease(k.getKey());
```

Comment:
Since `k.getKey()` returns `io.netty.buffer.ByteBuf`, you could use the `ByteBuf.release()` instead? Curious, why is `ReferenceCountUtil.safeRelease()` required here?

Reply:
It is a netty same release method that doesnt throw even if it is unable to release, which isnt catastrophic and shouldnt throw.